Makefile.in

```diff
@@ -318,13 +318,20 @@ distprep: catman-do
 	$(AUTORECONF)
 	-rm -rf autom4te.cache
 
-install: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-sysconf host-key check-config
+install: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-iron-files install-sysconf host-key check-config
```

Comment:
So I'm going to change this. Travis is failing on the make install step. Some discrepancy between travis and my local docker setup. Changing the default install should make it all better.